README.md

```diff
@@ -67,12 +68,78 @@ iis_site { 'minimal':
 
 ### Types/Providers
 
+* [iis_application](#iis_application)
 * [iis_application_pool](#iis_application_pool)
 * [iis_feature](#iis_feature)
 * [iis_site](#iis_site)
 
 Here, include a complete list of your module's classes, types, providers, facts, along with the parameters for each. Users refer to this section (thus the name "Reference") to find specific details; most users don't read it per se.
 
+### iis_application
+
+Allows creation of a new IIS application.
+
+#### `ensure`
+
+Must be either `present` or `absent`. Present will ensure the application pool is created.
```

Comment:
present and absent are strings, so single quotes. While you're here, please change "will ensure" to "ensures"?

I assume this has a default of 'present'? This and the other parameters in this section should have default values, if they exist.